.github/workflows/main.yml

```diff
@@ -11,13 +11,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
-        os: ["macos-latest", "ubuntu-latest", "windows-latest"]
+        python-version: ["3.9", "3.10"]
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
         numpy_ver: ["latest"]
         include:
```

Comment:
Since this is an analysis package, I don't think its necessary to include the ops test, but it may be desirable for ecosystem consistency.

Reply:
Trying out the ops env, but the setup may be complicated enough that we drop it here.  Some of the required packages don't install there

Reply:
I recommend in the template we add a note that the ops test is only needed in data packages.

Reply:
moving forward without the operational tests. Cleaning up the code so we don't have untested lines.  If there is a demand for operational support for this code, we can add it back and test the environment more thoroughly.